### DISCUSSION TITLE
Home Assistant OS 11.4

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3057] [Announcements]
## Raspberry Pi

* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Remove rootfstype for Raspberry Pi 5 (#3037) @agners

## Home Assistant Yellow

* Add rtw89 firmware option missing in #3003 (#3043) @sairon

## Home Assistant Green

* Revert kernel patch causing failures on CIFS share disk usage (#3051) @sairon
* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Linux: Update kernel 6.1.71 (#3046) @sairon

## Open Virtual Appliance

* Revert kernel patch causing failures on CIFS share disk usage (#3051) @sairon
* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Linux: Update kernel 6.1.71 (#3046) @sairon

## Generic x86-64

* Revert kernel patch causing failures on CIFS share disk usage (#3051) @sairon
* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Linux: Update kernel 6.1.71 (#3046) @sairon

## Hardkernel ODROID

* Revert kernel patch causing failures on CIFS share disk usage (#3051) @sairon
* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Linux: Update kernel 6.1.71 (#3046) @sairon

## ASUS Tinker

* Revert kernel patch causing failures on CIFS share disk usage (#3051) @sairon
* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Linux: Update kernel 6.1.71 (#3046) @sairon

## Khadas VIM Series

* Revert kernel patch causing failures on CIFS share disk usage (#3051) @sairon
* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Linux: Update kernel 6.1.71 (#3046) @sairon

## Generic aarch64

* Revert kernel patch causing failures on CIFS share disk usage (#3051) @sairon
* Add rtw89 firmware option missing in #3003 (#3043) @sairon
* Linux: Update kernel 6.1.71 (#3046) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.4'>Home Assistant OS 11.4</a>.</em>

**Comment by sabelosimelane** [comment]
Appreciate this update, thank you.

Waiting to see if anyone has installed it without any issues.

**Reply by Topslakr** [reply]
Ran the update on my 'Open Virtual Appliance' VM and it nuked my install. Broke nearly everything. All of my Tasmota, Zigbee, etc., items were unavailable.

Thankfully, I rolled back the ZFS array and am back on 11.3.

**Reply by sairon** [reply]
Unless the Linux bump from 6.1.70 to 6.1.71 was the cause, it's quite unlikely the reason for the breakage was the OS update, because there are no other differences between 11.3 and 11.4. But without any logs or more information provided we can only guess and if there was really something bad, another update could break your install again.

**Reply by Topslakr** [reply]
I kept a copy of the VM before I rolled back. I'll spin it up and see what I can find. Good advice!

**Comment by mweiss1965** [comment]
After Updating from 11.3 to 11.4 -> Nabu Casa Remote UI was broken.
There was no connection possible. After rolling back to 11.3 all is fine again.
Raspberry PI 4 with Sky Connect Module

**Reply by sairon** [reply]
Just as for the comment above, it's unlikely it's related to the OS version itself. Ideally please try upgrading again and check the Core logs for errors.

**Reply by mweiss1965** [reply]
I tried again and now it worked. No issues.
Something must have gone wrong with the first update
Thanks for that :-)

**Reply by LeChuck2909** [reply]
i have the same issue with HA OS running with vmware on windows. unfortunately a rollback to 11.3 did not fix it as well now.

**Comment by andyshinn** [comment]
Upgraded my Yellow and all appears good so far.

**Comment by derkorte** [comment]
Anyone here with pi5? 
I'm running the 11.4dev20231226 on pi5 8gb and can't find any response about updating to 11.4 is recommended or not.

**Reply by Stadtvilla35** [reply]
Yes after updating it doesnt boot anymore

**Reply by Stadtvilla35** [reply]
I'll try to flash the image directly

**Reply by derkorte** [reply]
That's why I asked before updating 😅
Let me know if you got it work with 11.4
For me the dev is very stable with clear logs and no errors or bugs in the add-ons.

**Reply by Stadtvilla35** [reply]
Ok now its working fine. This is how it works for me:
- Make a Backup from existing 11.3
- Loading Image from actually 11.4
- https://github.com/home-assistant/operating-system/releases/download/11.4/haos_rpi5-64-11.4.img.xz
- make a a SD from that image with balenaetcher
- start Pi5 with that SD card and restore from Backup from 11.3
- after restoring its still at 11.4
- making updates hacks

I thing it's all about that changes
- Remove rootfstype for Raspberry Pi 5 (https://github.com/home-assistant/operating-system/pull/3037) @agners
with Remove rootfstype for Raspberry Pi 5

Because when i first update Pi doenst booting.
LED indicator green, than network connection led lights up green, than led indicator switches to red. Than all lights of. This is in a continious loop.

**Reply by roby1o** [reply]
I have similar problem: rpi5, 11.4 OS and Supervisor 2023.2.1. : every time i reboot system I get Kernel panic error e can't mount filesystem so pi5 go in boot loop and the only thing i can do is fresh image on sd.

I can live days without rebooting, but i was trying to fix another problem that is my Add-on start but HA thing they are not so the front end doesn't open, but if i click "Start" my supervisor logs says "Add-on already started".
